draft-ietf-ohai-ohttp.md

```diff
@@ -757,20 +784,20 @@ RECOMMENDED; see {{server-responsibilities}}.
 To achieve the stated privacy goals, the Oblivious Relay Resource cannot be
 operated by the same entity as the Oblivious Gateway Resource. However,
 colocation of the Oblivious Gateway Resource and Target Resource simplifies the
-interactions between those resources without affecting client privacy.
+interactions between those resources without affecting Client privacy.
 
 As a consequence of this configuration, Oblivious HTTP prevents linkability
 described above. Informally, this means:
 
-1. Requests and responses are known only to clients and targets in possession
+1. Requests and responses are known only to Clients and servers in possession
    of the corresponding response encapsulation key and HPKE keying material.
    In particular, the Oblivious Relay knows the origin and destination of an
    Encapsulated Request and Response, yet does not know the decrypted
    contents. Likewise, targets know only the Oblivious Gateway origin, i.e.,
-   the relay, and the decrypted request. Only the client knows both the
+   the relay, and the decrypted request. Only the Client knows both the
    plaintext request and response.
```

Comment:
```suggestion
   plaintext request and response and its identity.
```

Reply:
What is "its" in this context?  (Also, and and and is a bit strange.)

Reply:
"it" is the client here. We say above that the gateway sees both plaintext request and response, but then here we say "only the client sees plaintext request and response," which is clearly a contradiction. (It's not _only_ the client.) So adding the fact that the client knows its own identity distinguishes its view from that of the gateway.

Reply:
Ok, maybe instead,

```suggestion
   plaintext request. No other entity that can see these plaintext messages can attribute them to the client.
```

Or drop the tweak. I worry that we might need to qualify this further: this somewhat depends on there being multiple concurrent users of the same relay making similarly sized requests. That sort of thing.

Reply:
That's better, but still contradictory. The text still says that "_only_ the client knows both the plaintext request and response," which is false since both the gateway and target see the plaintext request and response. Maybe we just drop "Only the client..." and instead write "No entity that can see plaintext messages can attribute them to the client which generated them"?

Reply:
Sold.